package.json

```diff
@@ -16,12 +16,13 @@
         "js-yaml": "3.13.1",
         "jsforce": "1.9.3",
         "json-stable-stringify": "1.0.1",
-        "runtime-plugin-manager-clone": "0.1.0",
+        "json2csv": "^6.0.0-alpha.2",
```

Comment:
Should we remove ^ ?

Reply:
Agreed - let's lock it to the version we test with

Reply:
Fixed.